receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -513,6 +514,13 @@ func (prw *prometheusRemoteWriteReceiver) processHistogramTimeSeries(
 				hist.SetAggregationTemporality(pmetric.AggregationTemporalityCumulative)
 			}
 			metricCache[metricIDHash] = histMetric
+			switch ts.Metadata.Type {
+			case writev2.Metadata_METRIC_TYPE_HISTOGRAM:
+				histMetric.Metadata().PutStr(prometheus.MetricMetadataTypeKey, "histogram")
+			case writev2.Metadata_METRIC_TYPE_UNSPECIFIED:
+				histMetric.Metadata().PutStr(prometheus.MetricMetadataTypeKey, "unknown")
+			default:
+			}
```

Comment:
Maybe the default case should return an error? This because we are talking about a not listed case/

Reply:
What do you think if we add a comment under the default option with this info? Saying that as we are only dealing with histograms inside the `processHistogramTimeSeries` we don't need to define a behavior for this option.